src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -7,35 +7,66 @@
 import java.util.Map;
 
 import CommandingOfficers.Commander;
+import Terrain.GameMap;
+import UI.COStateInfo;
 
 public class CommanderOverlayArtist
 {
-  private static String overlayFundsString = "FUNDS     0";
-  private static int previousOverlayFunds = 0;
+  public static final int OVERLAY_WIDTH = 97;
+  public static final int OVERLAY_HEIGHT = 20;
+
   private static int animIndex = 0;
   private static long animIndexUpdateTime = 0;
   private static final int animIndexUpdateInterval = 8;
   
   private static Map<String, Sprite> activeAbilityTextSprites = new HashMap<String, Sprite>();
 
+  /**
+   * Draws an overlay for all COs passed in, left-justified, with status text and optionally highlights one CO
+   */
+  public static BufferedImage drawAllCommanderOverlays(Commander[] coList, GameMap drawableMap, int drawingWidth, int drawingHeight, Commander coToHighlight)
```

Comment:
When it switches to drawing all overlays, and an ability is primed, the pip pulses much more slowly than it does when only a single CO overlay is being shown.